src/locales/fr_FR/fr_FR.json

```diff
@@ -0,0 +1,449 @@
+{
+    "accessibility": {
+        "camera_view": "Vue de la caméra",
+        "close_sdk_screen": "Fermer l'écran de vérification de l'identité",
+        "country_select": "Choisissez le pays",
+        "cross_device_verification": "Étapes requises pour continuer la vérification sur votre mobile",
+        "dismiss_alert": "Ignorer l'alerte",
+        "document_types": "Documents que vous pouvez utiliser pour vérifier votre identité",
+        "replay_video": "Rejouer la vidéo",
+        "selfie_video_actions": "Actions pour enregistrer une vidéo de votre visage",
+        "shutter": "Prendre une photo",
+        "start_recording": "Commencer l'enregistrement",
+        "stop_recording": "Arrêter l’enregistrement"
+    },
+    "back": "retour",
+    "bank_building_society_statement": "Bank or building society statement",
+    "benefit_letters": "Benefits Letter",
+    "cancel": "Annuler",
+    "capture": {
+        "bank_building_society_statement": {
+            "front": {
+                "instructions": "Fournissez la page entière du document pour de meilleurs résultats",
+                "sub_title": "Must be issued in the 1last 3 months1",
```

Comment:
did we stop using sub_title and webcam? Asking because they are in english

Reply:
Just noticed this. The format looks a bit odd also for the last 2 words `1last 3 months1`

Reply:
I think it's because we are only supporting the Proof of Address feature in English, so those strings are not translated.

Reply:
Yes, the formatting is wrong. I will see how we can flag them and get them fixed.